master/static/srv/delete_model.sql

```diff
@@ -1,13 +1,23 @@
-WITH m AS (
-  SELECT * FROM models
+WITH myuser AS (
```

Comment:
optional: I believe `a.CurrentUser` already gives you a user instance (so you know id and `admin` on go side).
so potentially you don't need this extra `WITH clause`, and just have `WHERE (user_id = $2 OR $3 IS TRUE)`, where $2 is current_user.id and $3 is current_user.admin.